.circleci/config.yml

```diff
@@ -76,6 +76,16 @@ step-library:
         command: |
           xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=13.4.1,name=iPhone 8 Plus' -project MapboxNavigation.xcodeproj -scheme Example clean build | xcpretty
 
+  - &trigger-metrics
+      run:
+        name: Trigger metrics
+        command: |
+          if [ -n "${MOBILE_METRICS_TOKEN}" ]; then
```

Comment:
How do we provide `MOBILE_METRICS_TOKEN` here? Is it environment variable on CircleCI?

Reply:
Correct